src/extension.ts

```diff
@@ -10,6 +10,15 @@ import * as path from 'path';
 import * as Url from 'url';
 import * as fs from 'fs';
 
+function checkValidEditor() : boolean {
+    if (vscode.window.activeTextEditor === undefined) {
```

Comment:
It just text-edits the document, but doesn't enable extensions.  Try the "embedded" version of MetalRoughSpheres.gltf.